tests/unit/nupic/research/sp_learn_inference_test.py

```diff
@@ -138,6 +140,7 @@ def _runLearnInference(self,
       print "Test succeeded"
 
 
+  @unittest.skip("Currently fails due to switch from FDRCSpatial2 to SpatialPooler.")
```

Comment:
Please update the skip comment to elaborate on why the switch causes this test to fail (as you said, the new SP doesn't have explicit methods to get inhibition).